webapp/src/views/organizations/billing/Subscriptions/common/Plan.tsx

```diff
@@ -1,26 +1,27 @@
 import { styled } from '@mui/material';
 
-export const StyledPlan = styled('div')`
+export const Plan = styled('div')`
```

Comment:
I thought we'll have this convention to include `Styled` prefix

Reply:
I just don't feel, if it's good idea for exported common components. The user doesn't care if it's "real" component or styled component, right?